skimage/morphology/tests/test_max_tree.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
-from skimage.morphology import max_tree, area_closing, area_opening
-from skimage.morphology import max_tree_local_maxima, diameter_opening
-from skimage.morphology import diameter_closing
+from skimage.morphology.max_tree import (area_closing, area_opening,
```

Comment:
Same applies here.

Reply:
Okay. I will revert these changes and explore the approach suggested [here](https://github.com/scikit-image/scikit-image/pull/6983).